src/internal/utils/focus-svg-element.ts

```diff
@@ -2,7 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 export default function focusSvgElement(element: SVGElement) {
   if (element.focus) {
-    element.focus();
+    // Using preventScroll here fixes a bug in Safari that causes the page to scroll to the top
+    // when hovering over a pie chart with no visible labels.
+    element.focus({ preventScroll: true });
```

Comment:
Since it is a common utility, it could be used in cases where the scroll is desired.

This is why it would be better to proxy this option via argument

Reply:
P.S. there is also an interesting else-condition which can be cleaned up

Reply:
Good point. Since we don't really need the IE11 condition anymore, we could just get rid of this utility entirely.

What do you think @just-boris @pan-kot ?